// Copyright 2024 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";
option optimize_for = LITE_RUNTIME;

package mojo_base.test;

// A simple test proto with a string in it.
message TestMessage {
  optional string test = 1;
}

// Another simple test proto with a string in it.
message TestMessage2 {
  optional string test = 1;
}

// Another simple test proto without a string in it.
message TestMessage3 {
  optional int32 test = 1;
}
